PeerioTechnologies/peerio-desktop
# fix: handle opening DM with deleted user

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/15413/desktop-deleted-accounts-broken
https://github.com/PeerioTechnologies/peerio-icebear/pull/340

#### Testing instructions  
* open a DM with a deleted user
  * if you have a message history with that user, your message history should open in a readOnly chat
  * if you don't have a message history with that user, a readOnly chat will be created

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
